ibc-primitives/src/types/timestamp.rs

```diff
@@ -26,10 +26,12 @@ pub const ZERO_DURATION: Duration = Duration::from_secs(0);
 ///  It is also encoded as part of the
 /// `ibc::channel::types::timeout::TimeoutTimestamp` type for expressly keeping
 /// track of timeout timestamps.
+#[cfg_attr(feature = "arbitrary", derive(arbitrary::Arbitrary))]
```

Comment:
how about we `impl Arbitrary for Timestamp` directly. It seems easier that way.

Reply:
good call! Done in [80f89a1](https://github.com/cosmos/ibc-rs/pull/1390/commits/80f89a1caa998a9da88838b392e7cd38584472a9)